XRTK-Core/Packages/com.xrtk.core/Services/MixedRealityToolkit.cs

```diff
@@ -403,7 +403,10 @@ private void InitializeServiceLocator()
                 {
                     if (CreateAndRegisterService<IMixedRealityFocusProvider>(ActiveProfile.InputSystemProfile.FocusProviderType))
                     {
-                        RegisterServices(ActiveProfile.InputSystemProfile.ControllerDataProvidersProfile.RegisteredServiceConfigurations);
+                        if (ActiveProfile.InputSystemProfile.ControllerDataProvidersProfile != null)
```

Comment:
I probably need to pull this out and validate that all the other systems don't require the same check (They likely do)

Reply:
Looked into this some more. The Input System profile is a unique case where we actually have a seperate profile for our registered data providers. Might update the other systems later to follow suit. I think I mainly did it to make sure that there was enough space in the inspector without all the providers taking up all the room, so I put them into a dedicated profile and inspector.